release/archives/build.gradle

```diff
@@ -78,6 +85,10 @@ subprojects {
         }
     }
 
+    tasks.withType(Zip) {
+        enabled false
+    }
```

Comment:
This disables the tasks to build `.zip` distributions. I assume those are not needed?

Reply:
We do not build `zip` archives presently.